src/drivers/amqp_interop/Queue.php

```diff
@@ -305,6 +305,10 @@ public function listen()
 
                 $this->redeliver($message);
 
+                if (function_exists('pcntl_signal_dispatch') && $consumer instanceof \Enqueue\AmqpExt\AmqpConsumer) {
```

Comment:
Why `\Enqueue\AmqpExt\AmqpConsumer`?
We have interop driver - `Interop\Amqp\AmqpConsumer`.